scripts/reset_test_db.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Make sure to give this script execution permission (chmod +x reset_test_db.sh)
+
+# Check if the script is run as root (sudo)
+if [ "$EUID" -ne 0 ]; then
+  echo "Please run this script with sudo."
+  exit 1
+fi
+
+# Define the container name
```

Comment:
you can remove the excessive comments, too :)

Reply:
Fixed!